pages/about.mdx

```diff
@@ -214,4 +228,8 @@ This site exposes a public JSON API powered by [Next.js API routes](https://next
     padding: 0 0.25em;
     border-radius: 0.125em;
   }
+
+  pre code {
```

Comment:
Small fix. The padding is for inline code, but added funny padding on code blocks.

Reply:
Nice